lightpath/controller.py

```diff
@@ -527,8 +555,59 @@ def get_device(self, device: NodeName) -> Device:
         -------
         Device
             requested device
+
+        Raises
+        ------
+        DeviceLoadingError
+            if the requested device is in the facility but cannot be loaded
```

Comment:
Is this docstring true? It looks like the new behavior is to slot in a mock device on failure

Reply:
Ah you're right.  I took this out after adding the mock devices.  I'll fix that.